### DISCUSSION TITLE
Synchronization with Desktop Clients ?

### DISCUSSION THREAD

**Discussion by thomascrisan** [mtotschnig/MyExpenses#1356] [General]
Has there been any thought to synchronizing MyExpenses with a desktop client?  I use a desktop package called Moneydance written by a small company in Scotland.  It isn't open source, but it also isn't tied to large financial organizations.

The Desktop client is comprehensive; the Android application, not so much, as it's a method of updating registers and not much more.

https://infinitekind.com/moneydance

**Comment by mtotschnig** [comment]
Originally, My Expenses was also designed as a companion to Desktop finance applications (like Grisbi and GnuCash), which explains the "Export and reset" functionality: It allowed you to keep on your phone only the transactions that were not yet exported for use in these other tools.
Now I understand that you are asking for more than just registering data. I do not think this is realistic at the moment, since it would require a tight collaboration between the two projects with respect to the exchange protocol. 
If you are using a Windows system, you can run My Expenses via its Android Subsystem. It can be installed via the Windows store.
